src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/BsnesApi.cs

```diff
@@ -115,10 +115,10 @@ public BsnesApi(string dllPath, CoreComm comm, IEnumerable<Delegate> allCallback
 			{
 				Filename = "bsnes.wbx",
 				Path = dllPath,
-				SbrkHeapSizeKB = 14 * 1024,
-				InvisibleHeapSizeKB = 4,
-				MmapHeapSizeKB = 110 * 1024, // TODO: check whether this needs to be larger; it depends on the rom size
-				PlainHeapSizeKB = 0,
+				SbrkHeapSizeKB = 12 * 1024,
+				InvisibleHeapSizeKB = 136 * 1024, // TODO: check whether this needs to be larger; it depends on the rom size
```

Comment:
This comment has changed positions now, which I don't think is right, considering roms are still saved as vector, which uses `new` internally (see for examples bsnescore.cpp line 204).

Reply:
Ah, right. I guess it's more technically being double loaded here, as copies would be being placed in invisible heap. Not ideal but w/e.

Reply:
Why would copies be placed in the invisible heap? And where do they come from?